crates/rolldown_binding/src/utils/normalize_binding_options.rs

```diff
@@ -123,7 +124,7 @@ pub fn normalize_binding_options(
 
   let mut module_types = None;
   if let Some(raw) = input_options.module_types {
-    let mut tmp = HashMap::with_capacity(raw.len());
+    let mut tmp: FxHashMap<_, _> = FxHashMapExt::with_capacity(raw.len());
```

Comment:
I guess so. Instead of returning `Self` in `with_capacity`, we could return `FxHashMap` directly since the trait is only gonna be used on `FxHashMap`.

```rust
pub trait FxHashMapExt<K, V> {
  fn with_capacity(capacity: usize) -> FxHashMap<K, V>;
}
```

Should works

Reply:
I now tried that but it seems it doesn't work.
The error was

> cannot call associated function on trait without specifying the corresponding `impl` type cannot call associated function of trait

Reply:
`FxhashMap::with_capacity` just works.  No need to use `FxHashMapExt:: with_capacity`.